modules/manage/pages/audit-logging/audit-log-samples.adoc

```diff
@@ -0,0 +1,531 @@
+= Sample Audit Log Messages
```

Comment:
Please revise either the navtree entry or the heading such that they match. Right now, the navtree has "Audit Log Message Examples", which is not the same as this heading.